### DISCUSSION TITLE
Jellyfin dont recognize files on NFS mounted share

### DISCUSSION THREAD

**Discussion by drm-gith** [tteck/Proxmox#1631] [Help Needed]
A clear and concise description of the issue.
Mounted a NFS share on a path inside LXC Jelly container on proxmox. The mount is good, i can navigate and see the files via terminal, but the Jellyfin aplication dont recognize the folders mounted on the path. I tried change the LXC folder permissions, but dont work. Tried make a user with the same UID e GUID on the TrueNAS, where the NFS share is configured, but again dont work.
I also activate the NFS option on proxmox for that LXC, and install with a privileged container. But nothing is working.
Any advices ?
Thanks in advance

Please provide detailed steps to reproduce the issue.
Install LXC Jelly with script above (bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/jellyfin.sh)")
Install the nfs requeriments via terminal
Create a folder (neither on / or /mnt works)
mount the nfs share via /etc/fstab
acess jellyfin
try add a folder inside the mount on library

**Comment by Arc86** [comment]
Some time has passed, but I ran in the exact same issue. I was able to solve it in TrueNAS under NFS Share Advanced Settings

And set: 
Mapall User -> root
Mapall Group -> wheel

After that I was able to browse the folder with Jellyfin.

**Comment by yodaphone** [comment]
this is very easy to do

Mount the NFS Share in Proxmox Host / Create Storage under Datacenter (You can use the UI and say it contains snippets. it really doesnt matter)
you cannot map the root folder in a NAS so you may want to map a folder like for example "music"

create a directory inside the container
mkdir -p /mnt/nasmedia

On the host run the following command
pct set 100(whatever your container id is) -mp0 /mnt/pve/music,mp=/mnt/nasmedia

thats it. not reboot the container and you will be able to see the folder inside the container

**Reply by LoadingStill** [reply]
You have the same solution I found as well.  But when I used the update script “Proxmox VE LXC Updater” the config disappears.  Is there a better way to update these containers that will not lose this config?

**Reply by tteck** [reply]
The Proxmox VE LXC Updater script executes an `apt-get update && apt-get dist-upgrade`. It does **not** edit any config.

https://github.com/tteck/Proxmox/blob/661411d8e8b091183521ee709280079eeb7ea685/misc/update-lxcs.sh#L70

**Reply by LoadingStill** [reply]
Hmm that is what I thought.  I need to go check again tonight because it was causing a lot of issues for me this past week.  Could be my PVE is just borked from my tinkering.
Thank you for the reply.
